objects.go

```diff
@@ -636,7 +687,7 @@ type HostRecordIpv6Addr struct {
 	IBBase     `json:"-"`
 	Ipv6Addr   string `json:"ipv6addr,omitempty"`
 	Ref        string `json:"_ref,omitempty"`
-	Duid       string `json:"duid"`
+	Duid       string `json:"duid,omitempty"`
```

Comment:
The same as for 'mac' field.

Reply:
This was a new change that i did not have in my branch. I have removed it in my branch.